app/domain/audits/policies/allocated.rb

```diff
@@ -1,6 +1,8 @@
 module Audits
   module Policies
     class Allocated
+      NO_ONE = :no_one
```

Comment:
Not sure about this constant nor its relationship with the drop downs.

Reply:
What do you mean by "not sure"? As in, you want me to find-replace? Or delete it?